plugins/auth-backend/src/service/router.ts

```diff
@@ -36,14 +36,17 @@ export async function createRouter(
   const router = Router();
   const logger = options.logger.child({ plugin: 'auth' });
 
-  const baseUrl = `${options.config.getString('backend.baseUrl')}/auth`;
+  const appUrl = options.config.getString('app.baseUrl');
```

Comment:
Should make this `appOrigin`, i.e. parsing the url to extract the origin.

Reply:
Thanks, updated!

Reply:
@Rugvip was this the way you intended? did you mean to actually URL parse `app.baseUrl` and then extract only host and port?

Reply:
Yeah we need to actually parse it and extract the origin, i.e. `new URL('http://localhost:3000/my/path').origin`